test.js

```diff
@@ -19,6 +19,7 @@ describe('updateNotifier', () => {
 			callback: options.callback || null
 		};
 	};
+	const argv = process.argv.slice(0);
```

Comment:
I figured the original `argv` could be captured just once before running the specs. But I am honestly not 100% that that's what happens.
Will update.